build.gradle

```diff
@@ -30,7 +30,7 @@ plugins {
   id "com.github.sherter.google-java-format" version "0.9"
   id "net.ltgt.errorprone" version "2.0.1" apply false
   id "com.github.johnrengelman.shadow" version "6.1.0" apply false
-  id "com.github.nbaztec.coveralls-jacoco" version "1.2.5" apply false
+  id "com.github.kt3k.coveralls" version "2.12.0" apply false
```

Comment:
Had to switch the coveralls upload plugin to get things working with this new config